### PR TITLE
Homework 01032023

### PR DESCRIPTION
**Pull request by bogdanovoleg**

### CODE REVIEW

File: src/main/java/lv/acodemy/Homework/VariablesDataTypes1.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import java.util.Arrays;
+
+public class VariablesDataTypes1 {
+    public static void main(String[] args) {
+        byte myCurrentAge = 29;
+        byte workYears = 6;
+        byte mortgageYears = 20;
+        short monthlySalary = 1300;
+        short currentYear = 2023;
+        short BirthYear = 1993;
```
Comment (acodemylv) [inline src/main/java/lv/acodemy/Homework/VariablesDataTypes1.java]: incorrect naming

File: src/main/java/lv/acodemy/Homework/VariablesDataTypes1.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        long populationOfEurope = 746400000;
+        long mileageInKm = 180000;
+        char letter = 'Q';
+        char letter2 = 'A';
+        float iosVersion = 16.3f;
+        float endCurseDate = 11.04f;
+        double currencyEuroPound = 0.88;
+        double mortgageAnnualPercentage = 1.9;//%
+        boolean IsApproved = true;
+        boolean IsDenial = false;
```
Comment (acodemylv) [inline src/main/java/lv/acodemy/Homework/VariablesDataTypes1.java]: is prefix should start with lowercase letter

File: src/main/java/lv/acodemy/Homework/VariablesDataTypes1.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        boolean lastIosVersion = true;
```
Comment (acodemylv) [inline src/main/java/lv/acodemy/Homework/VariablesDataTypes1.java]: can be named as: isLastIOsVersion

**Review by acodemylv — CHANGES_REQUESTED** [review]

File: src/main/java/lv/acodemy/Homework/VariablesDataTypes1.java (hunk 1 above)
Comment (bogdanovoleg) [reply src/main/java/lv/acodemy/Homework/VariablesDataTypes1.java]: BirthYear should start with lowercase letter. Is there something else incorrect with those lines?

**Pull request merged**